pkg/controller/cache/managed.go

```diff
@@ -222,6 +223,31 @@ func (e *external) Delete(ctx context.Context, mg resource.Managed) error {
 	return errors.Wrap(resource.Ignore(elasticache.IsNotFound, err), errDeleteReplicationGroup)
 }
 
+type tagger struct {
+	kube client.Client
+}
+
+func (t *tagger) Initialize(ctx context.Context, mg resource.Managed) error {
```

Comment:
These `Initialize` functions get the tags provided in the spec and the external tags that Crossplane now sets and merges them together then updates the tags in the spec, do I have that right?

Reply:
Yes, that's what they do. Simply appending Crossplane tags to the existing ones.

Reply:
Awesome, thanks for the clarification :)